wp/plugin/wp.ml

```diff
@@ -170,7 +171,9 @@ let analyze_proj (ctx : Z3.context) (var_gen : Env.var_gen) (proj : project)
   let specs = fun_specs flags to_inline in
   let exp_conds = exp_conds_mod flags in
   let stack_range = set_stack flags in
-  let env = Pre.mk_env ctx var_gen ~subs ~arch ~specs
+  let op_interp = (if (flags.op_interp = []) then Pre.op_interp_default
```

Comment:
Can we move this computation for the value of op_interp into a separate function so we can reuse it in both the single and comparative cases without having to maintain both of them?

Reply:
Great idea.

Reply:
Actually, I'm going to instead remove the computation for the _non_-comparative case, since the only time this is used is in comparisons.

Reply:
Sounds good! If it's only used for comparisons, I think we should have a note about that in the README and docs for the flag.